onnxruntime/test/providers/checkers.cc

```diff
@@ -80,6 +80,10 @@ struct DefaultTolerance<MLFloat16> {
     if (provider_type == kDmlExecutionProvider) {
       return 0.005f;
     }
+    if (provider_type == kXnnpackExecutionProvider) {
+      // To allow tests like ConvTranspose_2D_Bias_1 to pass
+      return 0.05f;
+    }
```

Comment:
nit: given this is quite a large tolerance I'd prefer we set it on a per-test basis as needed, as using this for all tests may hide an issue. 

e.g. a simple operator like Add where only one floating point operation to calculate each output value is involved should never need this much tolerance. for something like a Conv or ConvTranspose which has more floating point operations and the order of those may vary it is a more reasonable to have a larger tolerance if a test requires it.